modules/ossm-exposing-a-gateway-to-traffic-outside-the-cluster-using-openshift-routes.adoc

```diff
@@ -39,20 +39,29 @@ spec:
     name: <gateway_name>
     weight: 100
   wildcardPolicy: None
-  # ...
 ----
 
-. Verify that httpbin service can be accessed from outside the cluster through the ingress router. Ensure that you set the `INGRESS_HOST` variable appropriately for the environment that your cluster is running in. 
+. Verify that `httpbin` service can be accessed from outside the cluster through the ingress router. Ensure that you set the `INGRESS_HOST` variable appropriately for the environment that your cluster is running in. 
+
+.. If the cluster runs on AWS, run the following command to set the `INGRESS_HOST` variable:
```

Comment:
Totally up to you but it might follow our guidelines more closely (and tell users the important action sooner) if you say to run a command at the end:
```suggestion
.. If the cluster runs on AWS,  set the `INGRESS_HOST` variable by running the following command:
```

If you do decide to go with that suggestion, please also apply it to the next substep

Reply:
done